aareg/src/main/kotlin/no/nav/helsearbeidsgiver/inntektsmelding/aareg/ArbeidsforholdLøser.kt

```diff
@@ -33,7 +35,9 @@ class ArbeidsforholdLøser(
                 it.requireKey(
                     Key.ID.str,
                     Key.IDENTITETSNUMMER.str
+                    //    Key.UUID.str
```

Comment:
Ikke kritisk, men for logging purpose - burde vi egentlig alltid ha med UUID i alle løsere? Finnes det tilfeller der dette ikke kommer eller kan utledes? Ikke veldig viktig akkurat i dette tilfellet, men la oss tenke på det..

Reply:
Vi trenger å ta en prat i forhold til alle disse. Grunnet til det er at den UUID ikke funker akkuratt som transaksjons ID. Samtidig vi har ID og i tilleg vi har forespoerselId. Så det begynner å bli skikkelig forvirende. Også jeg er litt usikkert generet at måtten UUID er definert per i dag man kan bruke det for transaksjonsID. Så Jeg tenker generelt vi trenger å ha en gjennogang på alle disse.